server/events/vcs/gitlab_client.go

```diff
@@ -49,25 +49,28 @@ func NewGitlabClient(hostname string, token string, logger *logging.SimpleLogger
 
 	// If not using gitlab.com we need to set the URL to the API.
 	if hostname != "gitlab.com" {
-		u, err := url.Parse(hostname)
+		//We assume the url will be over HTTPS if the user doesn't specify a scheme
```

Comment:
Need a space after the `//`: `//We` => `// We`. Golang comments recommend full sentences, so you need a period at the end.